Automattic/wp-calypso
# Goals Capture: Add import link to footer.

## Proposed Changes

- Add import link to the footer (for Plan B).
- If the user picks some goal and then clicks Import, we will also collect the goals they selected.

**When user clicks on the Import button, internally:**
- The `import` goal is automatically selected.
- The `goalsToIntent` map will short-circuit and picks `import` as the intent.

**When user clicks Back from import screen, internally:**
- The `import` goal is removed from selected goals.

**Other UI fixes:**
- Ensure there is enough whitespace at the bottom of the screen when the page is scrolled all the way down (on both desktop & mobile).

## Testing Instructions

* Go to `calypso.localhost:3000/setup/goals?siteSlug=yoursite.wordpress.com&flags=signup/goals-step`.
  * Check if everything looks good in desktop & mobile view.
* Turn on `localStorage.setItem( 'debug', 'calypso:analytics*' );`
    * Use the analytics logging in Console to check:
      * if the right goals are selected
      * if the right intents are selected.

<img width="1348" alt="2022-06-20_11-03-03" src="https://user-images.githubusercontent.com/1287077/174586529-3b601817-a502-4c5f-97f6-cf8770632473.png">

<img width="500" alt="2022-06-20_11-02-31" src="https://user-images.githubusercontent.com/1287077/174586608-23bc5b13-e6ee-4c06-a679-bdf04b5e93aa.png">
